docs/organization/integrations/feature-flag/launchdarkly/index.mdx

```diff
@@ -23,3 +25,27 @@ The LaunchDarkly integration is only available to organizations with a Business
 1. Navigate to **Settings > Integrations > LaunchDarkly**
 
 2. Follow the full [LaunchDarkly installation instructions](https://docs.launchdarkly.com/integrations/sentry).
+
+## Evaluation Tracking
+
+Sentry can track flag evaluations as they happen within your application.  Flag evaluations will appear in the "Feature Flag" section of Issue Details page as a table, with "suspect" flag predictions highlighted in yellow. Learn more about how to interact with feature flag insights within the Sentry UI by reading the [Issue Details page documentation](/product/issues/issue-details/#feature-flags).
+
+### Set Up Evaluation Tracking
+
+To set up evaluation tracking visit one of our supported languages documentation page:
+* [JavaScript](/platforms/javascript/feature-flags/)
+* [Python](/platforms/python/feature-flags/)
+
+## Change Tracking
+
+Sentry can track changes to feature flag definitions and report suspicious feature flag edits.
+
+### Install and Configure
+
+To enable the change tracking integration click the "Set Up Integration" button on the feature flag table contained within the issue details page. This will trigger a fly out which will guide you through the two step process.
```

Comment:
```suggestion
To enable the change tracking integration go to the issue details page and click the "Set Up Integration" button on the feature flag table. This will trigger a fly out which will guide you through the two step process.
```

put sentence in order of operation, visit then click.